tool/common_mark_stats.json

```diff
@@ -279,12 +279,12 @@
   "115": "fail"
  },
  "Hard line breaks": {
-  "605": "strict",
```

Comment:
I think you fixed these, too....looking at travis failures. May want to regenerate